app/download/views.py

```diff
@@ -36,6 +36,36 @@ def download_document(service_id, document_id):
     except ValueError:
         return jsonify(error="Invalid decryption key"), 400
 
+    try:
+        check_scan_verdict(service_id, document_id)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), MALICIOUS_CONTENT_ERROR_CODE
+    except ScanInProgressError as e:
```

Comment:
I think we should not prevent the download if the scan is still in progress, because we will get that error if the scan times out. Here's the timeout logic: https://github.com/cds-snc/notification-document-download-api/blob/main/app/download/views.py#L127
You will see it if you pull the latest from main @jzbahrai.